kubectl-cp: add page

mohammadrony opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md.

Sign the CLA before submitting a pull request or it will be closed after some time.
https://cla-assistant.io/tldr-pages/tldr
-->

### Checklist

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR contains at most 5 new pages.
- [x] The PR is authored by me, or has been human-reviewed if it was created with AI or machine translation software.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
- Reference issue: #

emmanuel-ferdman reviewed (commented):
Thanks for the PR on `kubectl`. The page uses both `{{container_path}}` and `{{container_file}}` for the same concept - we shouldn't mix. But, looking into `scp.md` and `docker-cp.md` pages, we should use `{{path/to/local_file}}` and `{{path/to/file_in_pod}}` formats (or similar). Also, I think the `--retries` option is very useful we should consider adding it.

mohammadrony commented:
Thanks for the feedback. I’ve updated the page to use consistent path placeholders (`{{path/to/local_file}}` and `{{path/to/file_in_pod}}`) and added an example for the `--retries` option. Please let me know if any further changes are needed.

mohammadrony commented on pages/common/kubectl-cp.md:
```diff
@@ -0,0 +1,28 @@
+
+- Copy a local file or directory to a pod:
+
+`kubectl cp {{path/to/local_file}} {{pod_name}}:{{path/to/file_in_pod}}`
```
Update to address absolute path for pod. And local path can be relative.

mohammadrony reviewed (commented):
Update file reference in pod to use absolute path.
